### DISCUSSION TITLE
[Projects Beta] Sum of all numbered values for custom fields

### DISCUSSION THREAD

**Discussion by lightningspirit** [community/community#9171] [Projects and Issues] [answered]
When adding a custom number field in Table, I'd like to see the sum of all values for that field near the column name so that I can implement the following scenarios:

1. Having a number field called **Hours** and grouping by **Milestone** to see the number of hours spent for each Milestone.
2. Having an iteration field called **Sprint**, a number field called **Story Points** and a view grouped by **Sprint** and see the number of **Story Points** already booked for that **Sprint**.

**Comment by hpvd** [comment]
there seems to be a similar discussion about "Sum of numbered values from custom field"
Please up vote this too:
https://github.com/github/feedback/discussions/5376

**Comment by Davidjwkinghorn** [comment]
1000000%!

One of my favorite things about Jira along with its other custom field stuff and "insights".

**Comment by Tenorotti** [comment]
A gazillion percent!!!!!  Are we there yet?

**Comment by lightningspirit** [comment, marked as answer]
On 27th September, the it was [announced](https://github.blog/changelog/2022-09-27-the-new-github-issues-september-27th-update/) the availability of displaying sum of numeric fields. Already implemented the scenarios described in the first comment successfully.
This issue should now be closed.
